### ISSUE TITLE
Add link to create issue from main web ui

### ISSUE THREAD

**Issue by pombredanne** [nexB/scancode.io#214]
This would be handy and could come pre-populated with some data from the current project such as the project name, pipeline(s) and download URL(s)

**Comment by tdruez** [comment]
Hi @hkaur008 yes of course :)
Make sure to checkout our new contribution section: https://scancodeio.readthedocs.io/en/latest/contributing.html

**Comment by hkaur008** [comment]
Okay, thank you

**Comment by adii21-Ux** [comment]
I don't think anyone is working on this should I work on this

**Comment by prathamaror** [comment]
can i start working on this i have some questions prior

**Comment by Bhavika-14** [comment]
Hey, I want to contribute in it but unable to create issue, please help me.

**Comment by Bhavika-14** [comment]
Hey, I want to contribute in it but unable to create issue, please help me.

**Comment by pombredanne** [comment]
@adii21-Ux @prathamaror @Bhavika-14 

Please post your questions here. Do not ask to ask ;)
Also may be this is not a big enough issue for three person? You should sync up.

**Comment by SidharajYadav** [comment]
how i can start contribute . 
can i set environment 
https://github.com/nexB/scancode.io/issues/210 #issues210

**Comment by streetcodec** [comment]
@pombredanne I imagine this issue is still open, please assign it to me!

**Comment by AyanSinhaMahapatra** [comment]
@amyra98  You don't need to be assigned for creating PRs for an issue, just us know how you're solving the issue and go for it.

**Comment by moazzam07** [comment]
@AyanSinhaMahapatra issues mean Github issues or somthing else?

**Comment by AyanSinhaMahapatra** [comment]
Yes ^

This has been attempted at https://github.com/nexB/scancode.io/pull/450 very minimally with https://github.com/nexB/scancode.io/blob/lf32-licensetext-devel/scantext/templates/scantext/includes/license_report.html but this is seriously lacking in a lot of areas:

1. the above is just for scantext (for license detection issues in scancode-toolkit), we can do this for other issues across scancode.io 
2. this is just pasting data in the URL itself and this has limitations (if fields are too large this does not work), maybe we can use the API/something else so there are no limitations on this
3. there should be potentially a minimal review page, where we see what data exactly we would be posting to the issue, so we can review we are not pasting anything we want to keep private onto github issues

@tdruez what do you think?

**Comment by moazzam07** [comment]
yeah we can do that

**Comment by moazzam07** [comment]
@AyanSinhaMahapatra what has to be in the issue?

**Comment by moazzam07** [comment]
@AyanSinhaMahapatra i have made a form i just wanted to know how can i submit it as an issue

**Comment by AyanSinhaMahapatra** [comment]
@moazzam07 is this pre-populated data that is available on the editable form?

See comment above: https://github.com/nexB/scancode.io/issues/214#issuecomment-1434543989 where we have a very minimal way to do this via github URLs directly but as mentioned above this does not work for large texts/data so might want to consider using the github API?

@pombredanne @tdruez what do you think?

**Comment by pombredanne** [comment]
@AyanSinhaMahapatra IMHo let's focus on a pure UI thing and it is OK if this does not work for large texts

**Comment by moazzam07** [comment]
@AyanSinhaMahapatra yes it is editable
@AyanSinhaMahapatra @pombredanne  so should i make it using github url?

**Comment by moazzam07** [comment]
hey @pombredanne @AyanSinhaMahapatra i have a modal to enter data
![image](https://user-images.githubusercontent.com/67731696/223672354-8ba18a49-5e6f-40eb-af32-a1e0e5e52a2d.png)
is this need any modification in fields

**Comment by AyanSinhaMahapatra** [comment]
@moazzam07 we could look at better examples of scancode.io bugs, see https://github.com/nexB/scancode.io/issues?q=is%3Aopen+is%3Aissue+label%3Abug 

We also want to have a field for errors probably?

**Comment by moazzam07** [comment]
@AyanSinhaMahapatra yeah I will add that field too and is there any issue in this form?

**Comment by moazzam07** [comment]
@AyanSinhaMahapatra yeah I will ad that field too

**Comment by moazzam07** [comment]
> @moazzam07 we could look at better examples of scancode.io bugs, see https://github.com/nexB/scancode.io/issues?q=is%3Aopen+is%3Aissue+label%3Abug

i didn't get this could you please elaborate
